app/assets/javascripts/sufia/featured_works.js

```diff
@@ -31,20 +31,37 @@ function dragAndDrop(selector) {
 }
 
 Blacklight.onLoad(function() {
-  $('a[data-behavior="feature"]').on('click', function(evt) {
+  $('#content-wrapper').on('click', 'a[data-behavior="feature"]', function(evt) {
     evt.preventDefault();
+    evt.stopPropagation();
     anchor = $(this);
     $.ajax({
        url: anchor.attr('href'),
        type: "post",
        success: function(data) {
-         anchor.before("Featured");
+         anchor.before('<a data-method="post" data-behavior="unfeature-page" href="'+anchor.attr('href')+'">Unfeature</a>');
          anchor.remove();
        }
     });
   });
 
-  $('a[data-behavior="unfeature"]').on('click', function(evt) {
+  $('#content-wrapper').on('click', 'a[data-behavior="unfeature-page"]', function(evt) {
```

Comment:
`#content-wrapper` appears on every page, but we only need to register this handler for pages that have the button on them, so can you add a narrower selector to the `show_actions` partial?